UE/Application/ISmsDb.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "Sms.hpp"
+#include <vector>
+
+namespace ue
+{
+
+class ISmsDb
+{
+public:
+    virtual ~ISmsDb() = default;
+
+    //wszystkie sms dla ue
+    //jeden specyficzny sms
+    //dodanie sms
```

Comment:
If we are merging to `main` then we consider this production-ready code, so I would remove unnecessary comments:
```suggestion
```